src/components/BarList/BarList.tsx

```diff
@@ -0,0 +1,126 @@
+import { CSSProperties, FC, HTMLAttributes, ReactNode } from 'react'
+import { composeClasses } from 'lib/classes'
+import { BarLabel } from './BarLabel'
+import { Height, Rounded, Size } from 'interfaces'
```

Comment:
Well, if you import with this route, when you try to build catch a error or warnings.

You need change the path to: `../../interfaces/types`